ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -536,46 +549,6 @@ def add_capacity(osd_size_capacity_requested):
             params=params.strip("\n"),
             format_type="json",
         )
-    lvspresent = localstorage.check_local_volume_set()
-    lv_set_present = localstorage.check_local_volume_set()
-    if lvpresent:
-        ocp_obj = OCP(
-            kind="localvolume", namespace=config.ENV_DATA["local_storage_namespace"]
-        )
-        localvolume_data = ocp_obj.get(resource_name="local-block")
-        device_list = localvolume_data["spec"]["storageClassDevices"][0]["devicePaths"]
-        final_device_list = localstorage.get_new_device_paths(
-            device_sets_required, osd_size_capacity_requested
-        )
-        device_list.sort()
-        final_device_list.sort()
-        if device_list == final_device_list:
-            raise ResourceNotFoundError("No Extra device found")
-        param = f"""[{{ "op": "replace", "path": "/spec/storageClassDevices/0/devicePaths",
-                                                 "value": {final_device_list}}}]"""
-        log.info(f"Final device list : {final_device_list}")
-        lvcr = localstorage.get_local_volume_cr()
-        log.info("Patching Local Volume CR...")
-        lvcr.patch(
-            resource_name=lvcr.get()["items"][0]["metadata"]["name"],
-            params=param.strip("\n"),
-            format_type="json",
-        )
-        localstorage.check_pvs_created(
-            int(len(final_device_list) / new_storage_devices_sets_count)
-        )
-    if lv_set_present:
-        check_pvs_present_for_ocs_expansion()
-
-    sc = get_storage_cluster()
-    # adding the storage capacity to the cluster
-    params = f"""[{{ "op": "replace", "path": "/spec/storageDeviceSets/0/count",
-                "value": {new_storage_devices_sets_count}}}]"""
-    sc.patch(
-        resource_name=sc.get()["items"][0]["metadata"]["name"],
-        params=params.strip("\n"),
-        format_type="json",
-    )
```

Comment:
Should we delete all these lines? If we use the old version with the local storage, Don't we need to execute these lines?

Reply:
yes we need at my code i have added it twice so I delete it

Reply:
Oh I see